File: 9613.cpp
#include <iostream>

int gcd(int, int);


int main()
{
		int b;
		std::cin >> b;
		while (b--)
		{
			int c[111];
			int n;
			std::cin >> n;
			for (int i = 0; i < n; i++)
			{
				std::cin >> c[i];
			}
			long long d = 0;
			for (int i = 0; i < n - 1; i++)
			{
				for (int j = i + 1; j < n; j++)
				{
					d += gcd(c[i], c[j]);
				}
			}
			std::cout << d << std::endl;

		}



	return 0;
}

int gcd(int a, int b)
{
	if (b == 0) return 0;
	else gcd(b, a % b);
}

File: 9613.py
#9613
def gcd(x, y):
    if y == 0:
        return x
    else:
        return gcd(y, x%y)
t = int(input())
for _ in range(t):
    a = list(map(int, input().split()))
    n = a[0]
    a = a[1:]
    ans = 0
    for i in range(0, n-1):
        for j in range(i+1, n):
            ans += gcd(a[i], a[j])
    print(ans)
